src/workers/cdc/primary.ts

```diff
@@ -574,6 +590,18 @@ const onPostChange = async (
       }
     }
 
+    if (
+      data.payload.after!.type === PostType.Poll &&
+      data.payload.after!.deleted
+    ) {
+      cancelEntityReminderWorkflow({
+        entityId: data.payload.after!.id,
+        entityTableName: getTableName(con, PollPost),
+        scheduledAtMs: 0,
+        delayMs: 0,
```

Comment:
you will have to calculate same `delayMs` as for `runEntityReminderWorkflow` since `delayMs` is part of the job id so only then it will invalidate correct job